### DISCUSSION TITLE
Make an API to load a single island

### DISCUSSION THREAD

**Discussion by mdaffa48** [BG-Software-LLC/SuperiorSkyblock2#324] [Denied Feature Requests] [answered]
I want to use the Advanced-SlimeWorldManager add-on and fortunately, I got it working, but this add-on is almost perfect. My goal is to create multiserver skyblock servers, just like what Hypixel did, basically, there will be 10-20 island per server and there are gonna be multiple servers and because of that I need somehow an API to load island dynamically and I want to completely disable islands loads on plugin startup.

**Comment by OmerBenGera** [comment, marked as answer]
> I want to use the Advanced-SlimeWorldManager add-on and fortunately, I got it working, but this add-on is almost perfect. My goal is to create multiserver skyblock servers, just like what Hypixel did, basically, there will be 10-20 island per server and there are gonna be multiple servers and because of that I need somehow an API to load island dynamically and I want to completely disable islands loads on plugin startup.

You cant do such thing, as data of SSB will not be synced across servers. Also, the features ypu want are not possible to be implemented, as the plugin doesnt track such information.

**Reply by mdaffa48** [reply]
What information are you talking about? By the way, will SuperiorSkyblock data sync If there are 2 or more servers run with the same database? (Of course the same island won't be spawned in each server)

**Reply by OmerBenGera** [reply]
> What information are you talking about? By the way, will SuperiorSkyblock data sync If there are 2 or more servers run with the same database? (Of course the same island won't be spawned in each server)

No, it wont

**Reply by mdaffa48** [reply]
Alright, thanks for clarifying it, I think you can close this discussion.
